src/calcium/qqbar.jl

```diff
@@ -1530,4 +1530,20 @@ end
 #
 ###############################################################################
 
+"""
+    algebraic_closure(::QQField)
+
+Return a field representing the algebraic closure of the field of
+rational numbers.
+
+# Examples
+
+```jldoctest
+julia> K = algebraic_closure(QQ)
+Field of algebraic numbers
+
+julia> sqrt(K(2))
```

Comment:
```suggestion
julia> QQBar = algebraic_closure(QQ)
Field of algebraic numbers

julia> sqrt(QQBar(2))
```
maybe change here as well for consistency

Reply:
I really don't like using `QQBar` like this, I wouldn't use it like that myself -- I'd rather use a "realistic" example, with a name like `K`. (besides, wouldn't that break CI if I did it now, while Nemo still provides and exports `QQBar`).